lib/faexport.rb

```diff
@@ -400,6 +400,222 @@ def ensure_login!
       end
     end
 
+    # GET /notifications/submissions.json
+    # GET /notifications/submissions.xml
+    # GET /notifications/submissions.rss
+    get %r{/notifications/submissions\.(json|xml|rss)} do |type|
+      ensure_login!
+      set_content_type(type)
+      from_id = params['from'] if params['from'] =~ ID_REGEX
+      cache("submissions:#{@user_cookie}:#{from_id}.#{type}") do
+        case type
+        when 'json'
+          JSON.pretty_generate @fa.new_submissions(from_id)
+        when 'xml'
+          @fa.new_submissions(from_id).to_xml(root: 'results', skip_types: true)
+        when 'rss'
+          results = @fa.new_submissions(from_id)
+
+          @name = "New submissions"
+          @info = "New submissions for #{results["current_user"]["name"]}"
```

Comment:
These should be `:current_user` and `:name`

Reply:
Fixed in e6204b7ec09c5f7b854a81efb42388bc60215a21